### PR TITLE
Api integration

### PR DESCRIPTION
**Pull request by ezgi-c**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,29 +1,67 @@
...
+    let incompleteCount = items.filter((item) => !item.complete).length;
+    setIncomplete(incompleteCount);
+    document.title = `To Do List: ${incomplete}`;
+    
+    let sortedList = items?.sort((a, b) => a.difficulty - b.difficulty) || [];
+    setItems(sortedList);
+    // linter will want 'incomplete' added to dependency array unnecessarily.
+    // disable code used to avoid linter warning
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```
Comment (lizkavalski) [inline src/App.jsx:52]: If you want, you can add `items` in here if your tests are taking a long time

**Review by lizkavalski — COMMENTED** [review]
If your tests are not passing locally in the try adding this to your package.json:
```'
  "jest": {
    "moduleNameMapper": {
      "axios": "axios/dist/node/axios.cjs"
    }
```
[here the link to the source](https://stackoverflow.com/questions/73958968/cannot-use-import-statement-outside-a-module-with-axios)

**Comment by ezgi-c** [comment]
> 

Thank you!

File: src/App.jsx (hunk 1 above)
Comment (ezgi-c) [reply src/App.jsx:52]: Thank you!

**Pull request merged**